[SYCL][Doc] Fix AOT-related errors in SYCLCompilerAndRuntimeDesign.md (#767)
intel/llvm: sycl/doc/SYCLCompilerAndRuntimeDesign.md

architecture).  User can specify desired code format using the target-specific
option mechanism, similar to OpenMP.

> `-Xsycl-target-backend=<triple> "<arg>,<arg>"`

Reviewer: Argument is "<arg> <arg>" (not csv)

Reviewer 2: Why there are two args? There can be any number of them

Author: This was intended as "any number of args". Will fix the ambiguity and the format.

For example, to support offload to Gen9/vISA3.3, the following options would be used:

`-fsycl -fsycl-targets=spir64_gen-unknown-<os>-sycldevice -Xsycl-target-backend "-device skl"`

> The `<arg>` parameter is passed by the driver directly to the SYCL device

Reviewer 2: ```suggestion
The `-device skl` parameter is passed by the driver directly to the Gen device compiler backend.
```

Author: Thanks for the suggestions, applying soon

> compiler for the corresponding target w/o parsing it. For each target there is

Reviewer 2: ```suggestion
```

> some default code form which is generated in the absence of overriding via the

Reviewer 2: ```suggestion
```
